Brickficiency/Main.cs

```diff
@@ -1139,6 +1139,7 @@ public string GetPage(string url, bool login = false)
                 string pageSource;
                 //            StreamWriter swr = new StreamWriter(debugwebreqfilename);
 
+                // "(DateTime.Now > cookietime.AddMinutes(10))" is probably not an appropriate check as to wether to log in again.
```

Comment:
You're right, it's not.  Hopefully soon(tm) all logins will be handled by BricklinkSessionService and BricklinkLoginApi.